Zend/tests/enum/no-properties.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Enum disallows properties
+--FILE--
+<?php
+
+enum Foo {
+    public $bar;
+}
+
+?>
+--EXPECTF--
+Fatal error: Enums may not include member variables in %s on line %d
```

Comment:
```suggestion
Fatal error: Enums may not include properties in %s on line %d
```
We don't typically use "member variables".

Reply:
```
php-src/Zend/zend_compile.c
7097:           zend_error_noreturn(E_COMPILE_ERROR, "Interfaces may not include member variables");

php-src/tests/classes/interface_member.phpt
11:Fatal error: Interfaces may not include member variables in %s on line %d
```

I'd noticed that as well but saw it was based on the error message for interface - @nikic what are your thoughts on changing that error message for interfaces?

Reply:
Yes, changing the error message for interfaces sounds like a good idea.

Reply:
I'll adjust the message for both.